lib/platform/gitlab/index.ts

```diff
@@ -232,6 +232,7 @@ export async function initRepo({
   const repoConfig: RepoResult = {
     defaultBranch: config.defaultBranch,
     isFork: !!res.body.forked_from_project,
+    owner: repository.split('/')[0],
```

Comment:
```suggestion
    owner: res.body.namespace.path,
```
https://docs.gitlab.com/ee/api/projects.html#get-single-project

needs type change here: https://github.com/renovatebot/renovate/blob/6474dd49e53512746e547344b5e2804c358fde4c/lib/platform/gitlab/types.ts#L13